E/extrae/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env julia
+using BinaryBuilder, Pkg
```

Comment:
`Pkg` library not needed
```suggestion
using BinaryBuilder
```